src/lxc/start.c

```diff
@@ -174,6 +174,34 @@ static bool preserve_ns(int ns_fd[LXC_NS_MAX], int clone_flags, pid_t pid)
 	return false;
 }
 
+static int set_oom_score_adj(int oom_score_adj, pid_t pid)
+{
+	int ret;
+	FILE *f;
+/* 5 /proc + 21 /int_as_str + 14 /oom_score_adj + 1 \0 */
+#define OOM_PATH_LEN 45
+	char path[OOM_PATH_LEN];
+
+	ret = snprintf(path, OOM_PATH_LEN, "/proc/%d/oom_score_adj", pid);
+	if (ret < 0 || (size_t)ret >= OOM_PATH_LEN)
+		return -EFBIG;
+
+	f = fopen(path, "w");
+	if (!f) {
+		ERROR("Failed to open %s", path);
```

Comment:
Can you quote the path, please, aka `ERROR("Failed to open \"%s\", path)` it's something we now do with most strings in our log. :)